students/Ruohan/lesson04/kata.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+''' trigrams '''
+import string
+import random
+
+file_name = 'sherlock.txt'
+with open (file_name) as f_handle:
+# find the real content we need from the file
+    line_useful = ''
```

Comment:
line_useful will be all the lines eventually, yes? so maybe something like "text", "all_text" "all_lines" "good_lines", or even "useful_lines"....